services/venice-server/src/main/java/com/linkedin/venice/listener/grpc/VeniceReadServiceServer.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.venice.listener.grpc;
+
+import com.linkedin.venice.listener.grpc.interceptors.AuthorizationInterceptor;
+import io.grpc.Grpc;
+import io.grpc.InsecureServerCredentials;
+import io.grpc.Server;
+import io.grpc.ServerInterceptors;
+import java.io.IOException;
+
+
+public class VeniceReadServiceServer {
```

Comment:
You can either extend `VeniceGrpcServer` or have it as one of the fields in this class  to avoid code duplication